### ISSUE TITLE
160. Intersection of Two Linked Lists - Ambiguity on example 1

### ISSUE THREAD

**Issue by dog-dev-mel** [LeetCode-Feedback/LeetCode-Feedback#8598]
#### Your LeetCode username

My LeetCode username is 
dog-dev-mel

#### Category of the bug
- [x] Question
- [ ] Solution
- [ ] Language
- [ X] Missing Test Cases 

#### Description of the bug

In the example 1, I see many people discuss why "1" is not the intersection node.  Especially, when we only see the listA = [4,1,8,4,5], listB = [5,6,1,8,4,5]. Intuitively, we think the interaction node should be 1 rather than 4 if without the virtual graph diagram.

So it would be better to modify the explanation for the sentence  "The intersected node's value is 8 (note that this must not be 0 if the two lists intersect)." to "The Node with vaule 8 is selected/designed as the intersected node in example 1. if we choose the node with value 1 as the intersected node, the skipA and ship B should be 1 and 2, respectively."

#### Code you used for Submit/Run operation

```
// Two Sum
class Solution {
public:
    vector<int> twoSum(vector<int> &a, int s) {

    }
};
```

#### Expected behavior

I think this question is not hard in coding but some people are struggling with in the example 1 based on the responses from the discussion.

**Comment by LC-Pam** [comment]
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by 152334H** [comment]
duplicate of #8586

**Comment by LC-Pam** [comment]
Thank you for your time. We've used your feedback to update the problem. 
Added a note to Example 1 that the nodes with value 1 in listA and listB are different node references, and point to different locations in memory.
The main issue that in example 1, listA and listB have a common suffix of length 4: [1,8,4,5], and assume there is only one node with value 1.
Your LeetCode account has received 100 LeetCoins as the reward for this feedback. We appreciate your support!
